include/circt/Dialect/Datapath/DatapathOps.td

```diff
@@ -23,5 +23,85 @@ include "circt/Dialect/HW/HWTypes.td"
 class DatapathOp<string mnemonic, list<Trait> traits = []> :
     Op<DatapathDialect, mnemonic, traits>;
 
+//===----------------------------------------------------------------------===//
+
+// Compress an array of bitvectors to a smaller set of bitvectors (at least 2).
```

Comment:
Nit: why not add this to the description (thus also the documentation on the website) instead?